docs/install.md

```diff
@@ -4,12 +4,16 @@
 
 Hive is published to [OperatorHub](https://operatorhub.io/operator/hive-operator) weekly and this is the best method to install and use Hive if you do not need to build from source.
 
-  1. Create a 'hive' namespace.
-  1. In OpenShift web console navigate to Administrator perspective > Operators > OperatorHub.
-  1. Search for “hive” and select the OpenShift Hive operator.
-     1. Select the “alpha” update channel, install to a specific namespace, select the “hive” namespace just created, approval strategy automatic, and press Install.
-     1. You should now have a hive-operator pod running in the hive namespace.
-  1. Create a HiveConfig to trigger the the actual deployment of Hive. (can be done via the web UI with a couple clicks, or with oc apply)
+  1. Create a 'hive' namespace doing
+  `$ oc new-project hive`
+
+  2. Install the Hive Operator:
+  * In the [OpenShift web console](https://docs.openshift.com/container-platform/4.11/web_console/web-console-overview.html), navigate to [Administrator perspective](https://docs.openshift.com/container-platform/4.11/web_console/web-console-overview.html#accessing-the-administrator-perspective_web-console-overview) > Operators > [OperatorHub](https://docs.openshift.com/container-platform/4.11/operators/understanding/olm-understanding-operatorhub.html#olm-operatorhub-overview_olm-understanding-operatorhub).
```

Comment:
I think linking to documentation is okay and we do this in multiple places already (`grep -ir "docs.openshift.com" | wc -l` => `11`) but the links are instantly outdated as soon as a new version is released. The docs site supports specifying `latest` as the version in the URL so perhaps that is a good trade off assuming the URL path remains active.

Reply:
I wasn't sure how to refer to "latest" in the link. Done. Thanks!